scripts/data/experiment.json

```diff
@@ -156,4 +156,26 @@
     "endDate": "2016-02-02",
     "lastEdited": "2016-02-04",
     "structure": []
+}, {
+    "id": "test9",
+    "title": "Lookit Test",
+    "description": "It's a test",
+    "state": "Active",
+    "eligibilityCriteria": "age >= 3 years and age < 6 years",
+    "structure": [
+        {
+            "type": "lookit-overview",
+            "title": "Learning from Others: overview",
```

Comment:
Inconsistent use of title case- please change to capital-O:

"Learning from Others: Overview"